helm/aws-resolver-rules-operator/templates/podmonitor.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.monitoring.enabled }}
```

Comment:
I'm not blocking, this is just a question: why make it configurable? what's the use case where we want to disable it?